### ISSUE TITLE
508-defect-3 [COGNITION]: Names SHOULD not be ALL-CAPS

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#16197]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** Jennifer

## User Story or Problem Statement

As a user with dyslexia, I need names to be presented in Title Case so that I may read the name without struggle.

## Description

Not sure if the Veteran name is coming in with ALL-CAPS from the database, or if the data was entered in ALL-CAPS to match the design mock-up. The name **should** not be displayed in ALL-CAPS, as it creates issues for screen readers and people with dyslexia.

If possible, the database should be cleaned.

## Acceptance Criteria

- As a dyslexic user, I want to read the names of dependents on my account clearly.
- As a screen reader user, I want to understand the names of dependents on my account clearly.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: staging

## Steps to Recreate

1. Enter `https://staging.va.gov/decision-reviews/higher-level-review/request-higher-level-review-form-20-0996/veteran-information` in browser
2. Log in as user Nerea
3. Verify the name are ALL-CAPS

## Solution (optional)

1. In design mock-ups and sample data, do not use ALL-CAPS.

1. CSS text-transform won't fix this issue, and the information coming from the database should be cleaned.

## WCAG or Vendor Guidance (optional)

1. Some screen readers read capitalized text letter-by-letter. This occasionally even occurs when the HTML is in sentence-case and the CSS forces the capitalization. VoiceOver is especially persnickety about this.
1. "Text in all capital letters is more difficult to read for most people, with and without disabilities." Source: [Accessibility Requirements for People with Low Vision, Section 3.3.4](http://w3c.github.io/low-vision-a11y-tf/requirements.html#capitalization) All capital letters are harder to read (for everyone, but especially people with reading disabilities). The use of All Caps can reduce the readability of your text. Usability expert Jakob Nielsen has found that reading on screen can be around 25% slower than reading from paper, and reading All Caps can be a further 10% slower. It can be used for glance type uses, but it will still be an issue for screen reader users and people with dyslexia.
1. For people with dyslexia, ALL CAPS text may have no difference in shape: all the words are just one big rectangle. Sentence capitals have differentiated shapes based on the letters used. When producing information to be read by others, it is important to remember that up to 10% of your readers may have dyslexia. Dyslexia friendly text will have improved readability and better visual impact for all readers, but especially those with dyslexia.
1. If ALL CAPS must be used, consider using `aria-label="Sentence case words"` so screen readers will read the words rather than spelling out letters. Please validate with all screen readers available to dev. This will still leave issues for those with dyslexia.

## Screenshots or Trace Logs

![Screen Shot 2020-11-16 at 9 31 12 AM](https://user-images.githubusercontent.com/57469/99264654-c07c3f80-27ee-11eb-9308-4fa9eabf9e4d.png)

![Screen Shot 2020-11-16 at 9 35 00 AM](https://user-images.githubusercontent.com/57469/99265020-35e81000-27ef-11eb-916a-998c89f25fb2.png)

**Comment by Mottie** [comment]
The all-caps are how the data is stored in the mock data... we don't alter the user's name because of all the special rules on capitalization, e.g. `DiGeorge`, `O'Donnell`, `McDonald`, etc... also https://en.wikipedia.org/wiki/List_of_people_with_lower_case_names_and_pseudonyms

**Comment by jenstrickland** [comment]
Okeedoke! The database is dirty. Thanks for confirming. We've had some instances lately of folks ALL-CAPs-ing clean data, so wanted to be sure. Thanks for your patience.
